cmd/ignited/cmd/gitops.go

```diff
@@ -61,9 +67,24 @@ func NewCmdGitOps(out io.Writer) *cobra.Command {
 			}
 			if f.identityFile != "" {
 				var err error
+				// support ~ prefixes in the path
```

Comment:
Have you verified that this is necessary, since the shell will expand ~ before it is passed to the program? Unless this this config is read from a file, I don't see the need to do manual path expansion.

Reply:
it is needed, as we have the (non-expanded) part as a default